src/main/java/Duke.java

```diff
@@ -6,5 +9,101 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String lvlzero = "____________________________________________________________\n"
+                + " Hello! I'm Duke \n"
+                + " What can I do for you?\n"
+                + "____________________________________________________________\n";
+        System.out.println(lvlzero);
+
+        Tasks[] listOfTasks = new Tasks[100];
+        final String COMMAND_LIST_WORD = "list";
+        final String COMMAND_DONE_WORD = "done";
+        final String COMMAND_EVENT_WORD = "event";
+        final String COMMAND_DEADLINE_WORD = "deadline";
+        final String COMMAND_TODO_WORD = "todo";
+        final String COMMAND_BYE_WORD = "bye";
+        final String MESSAGE_LIST_TASKS = " Here are the tasks in your list:";
+        final String MESSAGE_ADD_TASK = " Got it. I've added this task:";
+        final String MESSAGE_MARK_DONE = " Nice! I've marked this task as done:";
+        final String MESSAGE_GOODBYE = " Bye. Hope to see you again soon!";
+        final String divider = "____________________________________________________________";
+        String description = null;
+        String by;
+        String date;
+
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            String firstCommandType = null;
+            String secondCommandType = null;
+            String inputLine = in.nextLine();
+
+            String[] commandType = inputLine.split(" ", 2);
+            if (commandType.length > 1) {
+                firstCommandType = commandType[0];
+                secondCommandType = commandType[1];
+            }
+            else {
+                firstCommandType = commandType[0];
+                secondCommandType = "";
+            }
+
+            switch (firstCommandType) {
+            case COMMAND_LIST_WORD:
+                System.out.println(divider);
+                System.out.println(MESSAGE_LIST_TASKS);
+                for (int i = 0; i < Tasks.getTotalTask(); ++i) {
+                    System.out.print("  " + listOfTasks[i].getTaskNumber()+ ". " + listOfTasks[i] + "\n");
+                }
+                System.out.println(divider);
+                break;
+            case COMMAND_BYE_WORD:
+                System.out.println(divider);
+                System.out.println(MESSAGE_GOODBYE);
+                System.out.println(divider);
+                break;
+            case COMMAND_DONE_WORD:
+                int ID = Integer.parseInt(secondCommandType) - 1;
+                listOfTasks[ID].markAsDone(true);
+                System.out.println(divider);
+                System.out.println(MESSAGE_MARK_DONE);
+                System.out.println("\t\t" + listOfTasks[ID].toString());
+                System.out.println(divider);
```

Comment:
These above lines could be refactored into one function because it is a code quality violation